custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/DashboardPage.kt

```diff
@@ -0,0 +1,37 @@
+package jces1209.vu.page.dashboard
+
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import java.net.URI
+
+abstract class DashboardPage(
+    protected val driver: WebDriver,
+    protected val uri: URI
+) {
+
+    open fun openDashboardsPage(): DashboardPage {
+        driver.navigate().to(uri.toURL())
+        return this
+    }
```

Comment:
A page shouldn't know how to open itself.
Consider this analogy:
```kotlin
val a = "Lorem ipsum"
val b = listOf("Lorem", "ipsum").joinToString(" ")
```
The "Lorem ipsum" `String` object is the same on both cases, but it doesn't know how it was created. There can be many ways of reaching the same content. Just like a content of a page can be reached via different routes.
E.g. by:
- opening by URI (like in this current code)
- getting redirected from a different URI
- navigating in via a link/button click
- being loaded via AJAX/XHR

I'd leave this choice to the higher layers. Let them navigate to a dashboard and construct a `DashboardPage` object.

Reply:
open dashboard moved into WorkOnDashboard